Avoid nesting

seblemaguer opened the pull request:
It is a bit patchy, but it aims to solves #137

jeremy-compostella reviewed (commented):
Could you make a nice commit message?

seblemaguer commented:
hello, sure I hope it is better :)

jeremy-compostella commented on org-msg.el:
```diff
@@ -1223,27 +1223,28 @@ MML tags."
-	    (unless (eq .style 'top-posting)
-	      (goto-char (point-max)))
-	    (insert .signature))
+	  (when (not (search-forward org-msg-options nil t))
```
(when (not expr) body) <=> (unless expr body)

jeremy-compostella reviewed (commented):
Could you briefly explain what it fixes ? a reference to the #156 in the commit message would also be helpful.

I added a code suggestion.

seblemaguer commented:
Updated with the code suggestion (thanks for it :))

I didn't quite understand this bit
> a reference to the https://github.com/jeremy-compostella/org-msg/pull/156 in the commit message would also be helpful.

I think you meant #137 so I modified it this way. If that is not what you meant, I can reword the commit
